examples/spring-boot-on-kubernetes-with-jib-custom-registry-example/src/main/resources/application.properties

```diff
@@ -1,2 +1,3 @@
 server.port=9090
 dekorate.jib.registry=quay.io
+dekorate.docker.enabled=false
```

Comment:
So, we need to disable explicitly conf other than the one we want to use?

Reply:
For now yes.
The next step is to prioritize build services that have been explicitly configured.
For example, if use configures jib and not docker jib gets priority.

Reply:
Got it!